content/docs/2.15/reference/events.md

```diff
@@ -0,0 +1,28 @@
++++
+title = "Events reference"
```

Comment:
This is just for Kubernetes events, CloudEvents is still under operate - What is reasoning here?

Reply:
I think we should have a table for it as well indeed.

However, given scenarios are different would it not be better to split this in to two tables to emphasize that?

Reply:
If that makes more sense, absolutely. My criterion would be: Does the user normally want the information in one table but not both? Is that what you mean by different scenarios?